src/translator/service.h

```diff
@@ -84,7 +84,7 @@ class Service {
   explicit Service(const std::string &config,
                    AlignedMemory modelMemory = AlignedMemory(),
                    AlignedMemory shortlistMemory = AlignedMemory())
-      : Service(parseOptions(config), std::move(modelMemory),
+      : Service(parseOptions(config, false), std::move(modelMemory),
```

Comment:
`/*validate=*/false`, since it's not obvious from reading here, if that's okay for you. ([ref](https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments))

Reply:
Done.